declarative-pipeline-migration-assistant/pom.xml

```diff
@@ -26,6 +26,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins.to-declarative</groupId>
       <artifactId>declarative-pipeline-migration-assistant-api</artifactId>
+      <version>${project.version}</version>
```

Comment:
No reason to have this dependency managed in the parent POM when this is the only consumer. Simpler to inline the version.